net.runelite.RuneLite.appdata.xml

```diff
@@ -21,7 +21,7 @@
     </screenshots>
 
     <releases>
-        <release version="2.4.0" date="2022-02-21"></release>
+        <release version="2.4.2" date="2022-03-14"></release>
```

Comment:
[2.4.2](https://github.com/runelite/launcher/releases/tag/2.4.2) was released on March 7th, not the 14th.
```suggestion
        <release version="2.4.2" date="2022-03-07"></release>
```